dashboard/db/migrate/20171204180201_create_featured_projects.rb

```diff
@@ -0,0 +1,9 @@
+class CreateFeaturedProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :featured_projects do |t|
+      t.string :project_id
+
+      t.timestamps
```

Comment:
Would it be helpful to also record the user id who marked it as a featured project?

Reply:
Recording the user id would be a good example of metadata that would justify having a separate table for this.